releases/katello/4.15/settings

```diff
@@ -0,0 +1,3 @@
+FULLVERSION='4.15.0'
+FOREMAN_VERSION='3.13'
+OSES="el8 el9"
```

Comment:
Since https://github.com/theforeman/theforeman-rel-eng/commit/0ee482d44812e73fac8b535da13abfe2d711e4ed it's better to leave it out since it's inherited from Foreman's config.
```suggestion
```